### PR TITLE
WFLY-14321 Replace 'JavaServer Pages|JSP' references in java doc with…

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-14321
(Replace "JavaServer Pages|JSP" references in java doc with corresponding Jakarta EE ones)
Link with: https://github.com/wildfly/wildfly/pull/13808

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
CC: @asoldano @scottmarlow

**Review by asoldano — CHANGES_REQUESTED** [review]
@RanabirChakraborty it looks like DeploymentOverlayCLITestCase is failing (I guess because of the a.jsp / a-replaced.jsp files' changes, but haven't checked). Can you please fix that ?

**Comment by RanabirChakraborty** [comment]
Thanks! @asoldano Done the changes :)

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
    <!-- 
       FacesServlet is main servlet responsible to handle all request. 
       It acts as central controller.
-      This servlet initializes the JSF components before the JSP is displayed.
+      This servlet initializes the JSF components before the Jakarta Server Page is displayed.
```
Comment (scottmarlow) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml:40]: I checked the [Jakarta Server Pages 3.0 spec](https://jakarta.ee/specifications/pages/3.0/jakarta-server-pages-spec-3.0.html) and they do not define the term `Jakarta Server Page`.  They do seem to loosely use the terms { `page`, `pages`, `Page`,  `Pages` } and still also refer to `JSP page`.  IMO, we should do our own thing, as the spec needs work.

How about changing to:
`This servlet initializes the Jakarta Server Faces components before the Jakarta Server Pages page is displayed.`
OR
`This servlet initializes the JSF components before the Jakarta Server Pages page is displayed.`

Comment (RanabirChakraborty) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml:40]: Okay! I'll do the changes.

Comment (RanabirChakraborty) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml]: Thanks! @scottmarlow Did the changes :)

Comment (scottmarlow) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml]: Thanks @RanabirChakraborty for the +10000 super effort!  Very much appreciated!

Comment (RanabirChakraborty) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/web.xml]: @scottmarlow I'm happy to be a part of it :) and anything for the community :)

**Comment by asoldano** [comment]
@scottmarlow are you fine with this being merged now?

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
> @scottmarlow are you fine with this being merged now?

@asoldano yes please do merge now.

**Pull request merged**